db/seeds/playbooks/crooked.seeds.rb

```diff
@@ -274,57 +274,21 @@
   #####
   # Gear
   #####
-  [
-    {
-      name: '.22 revolver',
-      harm: 1,
-      tag_list: %w[close reload small]
-    },
-    {
-      name: '.38 revolver',
-      harm: 2,
-      tag_list: %w[close reload loud]
-    },
-    {
-      name: '9mm',
-      harm: 2,
-      tag_list: %w[close loud]
-    },
-    {
-      name: 'Shotgun',
-      harm: 3,
-      tag_list: %w[close messy]
-    },
-    {
-      name: 'Hunting rifle',
-      harm: 2,
-      tag_list: %w[far loud]
-    },
-    {
-      name: 'Big knife',
-      harm: 1,
-      tag_list: %w[hand]
-    },
-    {
-      name: 'Baseball bat',
-      harm: 1,
-      tag_list: %w[hand]
-    },
-    {
-      name: 'Submachine gun',
-      harm: 2,
-      tag_list: %w[close reload area]
-    },
-    {
-      name: 'Assault rifle',
-      harm: 3,
-      tag_list: %w[close far area]
-    }
-  ].each do |gear_attrs|
+
+  # Array of associated gear ID's:
+  # 22 revolver (1-harm close reload small) -1
+  # .38 revolver (2-harm close reload loud) -2
+  # 9mm (2-harm close loud) - 3
+  # Shotgun (3-harm close messy) - 62
+  # Hunting rifle (2-harm far loud) - 46
+  # Big knife (1-harm hand) - 12
+  # Baseball bat (1-harm hand) - 8
+  # Submachinegun (2-harm close reload area) - 73
+  # Assault rifle (3-harm close/far area) - 5
+  [1, 2, 3, 62, 46, 12, 8, 73, 5].each do |gear_id|
```

Comment:
This list of ids will be different in each environment, especially production, which already gear loaded. Name and Harm would be a better way to associate this.

Reply:
We discussed this elsewhere, but just to have a record of what was decided in case we need to look back over it:
Unfortunately, searching by harm and name won't work, because we have entries that share name and harm.  The solution we came up with was to populate the through table through a reference in gear to what playbooks they belong in.  Initial plan is create a new column referencing the playbook names in the gear table, then rewrite the seed files to populate in that manner.

Reply:
Addressed in commits d886634 and f886cf9